package.json

```diff
@@ -90,7 +90,7 @@
     "typescript": "^3.7.0"
   },
   "dependencies": {
-    "mobx-react-lite": "2"
+    "mobx-react-lite": "2.0.6"
```

Comment:
Why depend on a specific version? I would rather not as it would be releasing a new version of this package when lite gets updated. If anything, it should be `>=2.0.6`.

Reply:
Valid question. I misinterpreted 2 to be a specific version and not a range